qt/scientific_interfaces/Indirect/IndirectFitDataModel.cpp

```diff
@@ -369,6 +369,15 @@ std::string IndirectFitDataModel::getExcludeRegion(FitDomainIndex index) const {
   return getExcludeRegion(subIndices.first, subIndices.second);
 }
 
+void IndirectFitDataModel::setExcludeRegion(const std::string &exclude,
+                                            FitDomainIndex index) {
+  if (m_fittingData->empty())
+    return;
+  auto subIndices = getSubIndices(index);
+  m_fittingData->at(subIndices.first.value)
```

Comment:
What would happen if we can't find `subIndices.first.value` in `m_fittingData`? Would it be worth adding a check before we do this?

Reply:
If there is no associated `TableDatasetIndex` for a `FitDomainIndex` then something has gone very wrong somewhere and we get a runtime error from getSubIndices.